js/apps/system/_admin/aardvark/APP/frontend/js/templates/modalApplicationMount.ejs

```diff
@@ -152,12 +152,15 @@
     <div class="tab-pane" id="zip">
       <div>
         <p>
-          Upload a ZIP-packed foxx application to ArangoDB.
+          1. Upload a ZIP-packed foxx application to ArangoDB.
           The ZIP file has to be created from the folder containing the manifest.json file of your Service, do not include the databases folder structure created by ArangoDB.
           This is also compatible with the download format for ZIP files on <a href="https://www.github.com">github.com</a>.
           Remember to change to a new version number in the manifest.json when uploading an update.
         </p>
         <table>
+        <p>
+          2. Or upload a single JavaScript-based file to ArangoDB.
```

Comment:
@hkernbach : shouldn't it be "JavaScript file" instead of "JavaScript-based file"?